.circleci/config.yml

```diff
@@ -145,20 +145,26 @@ jobs:
       - checkout
       - rn/ios_simulator_start:
           device: iPhone 15
+      - run:
+          name: Set Ruby 3.2.0
+          command: |
+            eval "$(rbenv init -)"
+            rbenv install -s 3.2.0
+            rbenv global 3.2.0
+            rbenv rehash
       - install-dependencies-ios-build
       - revenuecat/install-gem-mac-dependencies:
           cache-version: v2
           working_directory: examples/purchaseTesterTypescript
       - rn/pod_install:
+          cache: false
           pod_install_directory: examples/purchaseTesterTypescript/ios
           pod_install_command: bundle exec pod install --repo-update
-      - rn/ios_build:
-          build_configuration: Release
-          device: iPhone 15
-          derived_data_path: ~/DerivedData
-          project_type: workspace
-          project_path: examples/purchaseTesterTypescript/ios/PurchaseTester.xcworkspace
-          scheme: PurchaseTester
+      - run:
+          name: Build iOS
+          command: |
+            export RCT_NO_LAUNCH_PACKAGER=true \
+            && xcodebuild -workspace examples/purchaseTesterTypescript/ios/PurchaseTester.xcworkspace -destination 'platform=iOS Simulator,name=iPhone 15' -scheme 'PurchaseTester' -parallelizeTargets -configuration Release -derivedDataPath ~/DerivedData  -UseModernBuildSystem=YES
```

Comment:
THe actual `rn/ios_build` command pipes this through `xcpretty`, which isn't installed in the ruby version... I could try to install it, but felt easier to just do what the orb is doing internally by passing the command here.

Reply:
makes sense. I would maybe add a comment, but not a blocker